Gemfile

```diff
@@ -44,6 +44,7 @@ gem "tiktoken_ruby", "~> 0.0.6"
 gem "solid_queue", "~> 0.2.1"
 gem "name_of_person"
 gem "actioncable-enhanced-postgresql-adapter" # longer paylaods w/ postgresql actioncable
+gem "aws-sdk-s3", require: false
```

Comment:
I’m not familiar with require: false, I just looked it up and it sounds like when this is false the you need to explicitly set “require …” in the file that uses it. Is that the intent?

Reply:
Oh interesting. Maybe rails requires it explicitly internally. Just going based on the guides here: https://guides.rubyonrails.org/active_storage_overview.html#s3-service-amazon-s3-and-s3-compatible-apis